### DISCUSSION TITLE
Theme Hospital without dos emulators, I love you!

### DISCUSSION THREAD

**Discussion by Nephiro** [otya128/winevdm#1413] [Show and tell]
I have the original cd-rom of this game and it was a pain for me not being able to play it again without annoying dos emulators that most of the time bug the game, but finally Theme Hospital can shine again thanks to you and everyone made this tool possible!

Love you!

**Comment by Anixx** [comment]
Really works? I thought the graphical DOS mode is not supported.

**Reply by Nephiro** [reply]
Dunno about that, but I tried installing it using the CD and works (I remember that after win XP was impossible lanching the installation setup), I even launched the game and run much better than using dos emulator, so at the end I'm super happy

**Reply by cracyc** [reply]
That is a DOS or Win32 game.  I assume winevdm was just for the win32 version installer.

**Reply by Nephiro** [reply]
Nope, because even manually extracting the game from the cd rom and launching it (since with Theme Hospital you can) still doesn't work as it doesn't even start, like the installer. Instead since I'm using winevdm, I'm able to even play it.
Windows 7 cannot install it either, but if I remember correctly, extracting the game and launching it (like I just said) I was able to play it. Could be wrong since I tried this many many years ago

**Reply by Anixx** [reply]
Man, in your Task Manager screenshot, there is no Winevdm running. Your Theme Hospital process is running natively.

**Reply by Nephiro** [reply]
Ok, just to be sure, I uninstalled Winevdm, and it works. However it's strange, because I already tried extracting the game from the cd rom before and it didn't even start. Then maybe the installation installed some libraries on windows, that's the only explaination I could give. Because even in the system registry it didn't add any important keys

**Comment by Nephiro** [comment]
Just a few screenshot to prove what I'm saying

![immagine](https://github.com/user-attachments/assets/6764582c-be8b-41ca-85d9-af3c8046759c)

![immagine](https://github.com/user-attachments/assets/bb113df4-cdd5-40ca-be54-c8101e5d6b7d)

**Reply by Anixx** [reply]
Your game is running natively as a 32-bit application. Winevdm is not involved.
